sources/views/contacts.js

```diff
@@ -56,23 +56,31 @@ export default class Contacts extends JetView {
 		this.list.attachEvent("onItemClick", id => {
 			this.show(`./form?id=${id}`);
 		});
+		contacts.attachEvent("onAfterLoad", () => {
+			(url.length <= 1 && this.list.getFirstId()) ? (this.show("./form?id=" + this.list.getFirstId())) : 
```

Comment:
ternary operations are really hard to read in this situation. please rewtite them to `if..else` blocks (same for urlChange)

Reply:
Done)